pkg/ccl/changefeedccl/sink_webhook.go

```diff
@@ -81,15 +115,76 @@ type webhookSink struct {
 	eventsChans []chan []byte
 	inflight    *inflightTracker
 	retryCfg    retry.Options
+	batchCfg    batchConfig
+	// the i-th batchTracker should track and buffer events received by the
+	// i-th eventsChan. The guarantee here is per-key batching, so messages
+	// with the same key will be considered for the same batch.
```

Comment:
plans to change this to make it global with 1 buffer: https://github.com/cockroachdb/cockroach/issues/68656